Update Logging documentation for RHDEVDOCS-2619 and RHDEVDOCS-2627

Srivaralakshmi opened the pull request:
Aligned team: Dev Tools
OCP version for cherry-picking: `enterprise-4.7` and `4.8`
JIRA issues: [RHDEVDOCS-2619](https://issues.redhat.com/browse/RHDEVDOCS-2619), [RHDEVDOCS-2627](https://issues.redhat.com/browse/RHDEVDOCS-2627)
Preview pages: 
- For [RHDEVDOCS-2619](https://issues.redhat.com/browse/RHDEVDOCS-2619):
-- [Installing OpenShift Logging and Elasticsearch Operators](https://deploy-preview-30438--osdocs.netlify.app/openshift-dedicated/latest/logging/dedicated-cluster-deploying.html#dedicated-cluster-install-deploy)
-- "dedicated-cluster-logging.adoc" file changes:  No preview available, as the file is not yet published or included in any assembly.
- For [RHDEVDOCS-2627](https://issues.redhat.com/browse/RHDEVDOCS-2627):
-- [Viewing logging collector pods](https://deploy-preview-30438--osdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-collector.html#cluster-logging-collector-pod-location_cluster-logging-collector)

This pull request has been reviewed and approved by SME/QE: @anpingli and @lukas-vlcek

anpingli reviewed (commented):
/lgtm

lukas-vlcek commented:
/lgtm

Srivaralakshmi commented:
> /lgtm

Srivaralakshmi commented:
@anpingli and @lukas-vlcek Thanks a lot :)

pneedle-rh reviewed (commented):
@Srivaralakshmi I have now completed my peer review. Please let me know if you have any questions about my suggestions.

I realise that `logging/dedicated-cluster-logging.adoc` is not yet added to the topic map and that it will not be included in the build yet. But, I noticed that the file does not include a TOC. You can include `toc::[]` underneath the `context` definition to add a TOC. Additionally, a lot of the content in that assembly file could be moved to a new module that is added as an `include` line from the assembly.

pneedle-rh commented on modules/cluster-logging-collector-pod-location.adoc:
```diff
@@ -5,7 +5,7 @@
-You can use the `oc get pods  --all-namespaces -o wide` command to see the nodes where the Fluentd are deployed.
+You can use the `oc get pods  --all-namespaces -o wide` command to see all the pods and the nodes, and the details about where they are running.
 
 .Procedure
```
This single step in the procedure should use an unordered list. Please prepend `*` to the step.

Srivaralakshmi replied:
Done.

Srivaralakshmi commented:
> @Srivaralakshmi I have now completed my peer review. Please let me know if you have any questions about my suggestions.
> 
> I realise that `logging/dedicated-cluster-logging.adoc` is not yet added to the topic map and that it will not be included in the build yet. But, I noticed that the file does not include a TOC. You can include `toc::[]` underneath the `context` definition to add a TOC. Additionally, a lot of the content in that assembly file could be moved to a new module that is added as an `include` line from the assembly.

As this is not part of the PR scope, I am leaving it for now. But will inform Rolfe about it. Thanks!

Srivaralakshmi commented:
> @Srivaralakshmi I have now completed my peer review. Please let me know if you have any questions about my suggestions.
> 
> I realise that `logging/dedicated-cluster-logging.adoc` is not yet added to the topic map and that it will not be included in the build yet. But, I noticed that the file does not include a TOC. You can include `toc::[]` underneath the `context` definition to add a TOC. Additionally, a lot of the content in that assembly file could be moved to a new module that is added as an `include` line from the assembly.

I have put toc::[] for now. Moving the content - I am not addressing that for now as it it out of the PR scope. But I will inform @rolfedh about that. Thanks!

Srivaralakshmi commented on modules/cluster-logging-collector-pod-location.adoc:
```diff
@@ -5,7 +5,7 @@
 = Viewing logging collector pods
 
-You can use the `oc get pods  --all-namespaces -o wide` command to see the nodes where the Fluentd are deployed.
+You can use the `oc get pods  --all-namespaces -o wide` command to see all the pods and the nodes, and the details about where they are running.
```
Done.

Srivaralakshmi commented on modules/cluster-logging-collector-pod-location.adoc:
```diff
@@ -5,7 +5,7 @@
 = Viewing logging collector pods
 
-You can use the `oc get pods  --all-namespaces -o wide` command to see the nodes where the Fluentd are deployed.
+You can use the `oc get pods  --all-namespaces -o wide` command to see all the pods and the nodes, and the details about where they are running.
```
Done.

Srivaralakshmi commented:
@pneedle-rh As discussed, I and @anpingli reviewed the description. I have modified the content now as per his suggestion and got his LGTM for the same. I have incorporated almost all your comments. PTAL and if all is ok, kindly provide LGTM and merge. Thanks!

rolfedh commented:
Thanks @pneedle-rh @Srivaralakshmi, I've created https://issues.redhat.com/browse/RHDEVDOCS-2816 to track the suggested work.

anpingli reviewed (commented):
/lgtm

pneedle-rh commented on logging/dedicated-cluster-logging.adoc:
```diff
@@ -3,6 +3,8 @@
 = Configuring OpenShift Logging in {product-title}
 include::modules/common-attributes.adoc[]
 
+toc::[]
```
Thank you for adding this line. I checked in a local build and there is still no TOC, but that is now because there are no titles in the assembly. I realise this is out of scope for this PR, given that this assembly is not in the topic map and subsequently not included in the build.

Srivaralakshmi replied:
ok, thanks!

pneedle-rh reviewed (commented):
@Srivaralakshmi thank you for taking the time to review my suggestions. I have added some additional suggestions in relation to your updates. Please let me know if you have any questions. Thanks!

pneedle-rh commented:
> Thanks @pneedle-rh @Srivaralakshmi, I've created https://issues.redhat.com/browse/RHDEVDOCS-2816 to track the suggested work.

@rolfedh thank you!

Srivaralakshmi commented on modules/cluster-logging-collector-pod-location.adoc:
```diff
@@ -5,11 +5,12 @@
 = Viewing logging collector pods
 
-You can use the `oc get pods  --all-namespaces -o wide` command to see the nodes where the Fluentd are deployed.
+You can use the `oc get pod --selector component=fluentd -o wide -n openshift-logging` command to see all the Fluentd pods, and its details, for example, the node where each pod is running on.
```
Done.

Srivaralakshmi commented on modules/cluster-logging-collector-pod-location.adoc:
```diff
@@ -5,11 +5,12 @@
 .Procedure
 
-Run the following command in the `openshift-logging` project:
+* Run the following command in the `openshift-logging` project:
```
Done.

Srivaralakshmi commented:
> @Srivaralakshmi thank you for taking the time to review my suggestions. I have added some additional suggestions in relation to your updates. Please let me know if you have any questions.
> 
> Please also squash the 2 commits into 1.
> 
> Thanks!

@pneedle-rh Hi Paul, thanks for the review! I have done the requested changes and squashed the commits as well. Can you please have a look and merge it? Thanks again!

pneedle-rh commented on modules/cluster-logging-collector-pod-location.adoc:
```diff
@@ -5,11 +5,11 @@
 .Procedure
 
-Run the following command in the `openshift-logging` project:
+* Run the following command in the `openshift-logging` project to view the Fluentd logging collector pods and its details:
```
You could replace "its details" with "their details", since the sentence refers to multiple pods.

Srivaralakshmi replied:
Done.

pneedle-rh reviewed (commented):
@Srivaralakshmi thank you for your further updates. I have included one minor comment about one of the updates. Aside from that, this LGTM!

Srivaralakshmi commented:
> @Srivaralakshmi thank you for your further updates. I have included one minor comment about one of the updates. Aside from that, this LGTM!

@pneedle-rh Incorporated your comment. PTAL and if all is ok, kindly merge. Thanks :)

pneedle-rh commented:
@Srivaralakshmi thank you for taking the time to address each of my suggestions. This LGTM!

pneedle-rh commented:
/cherrypick enterprise-4.7

pneedle-rh commented:
/cherrypick enterprise-4.8
